code/modules/religion/religion_sects.dm

```diff
@@ -213,7 +216,107 @@
+	altar_icon_state = "convertaltar-white"
+	alignment = ALIGNMENT_GOOD
+	convert_opener = "\"A good, honourable crusade against evil is required.\"<br>\
+	Your diety requires fair fights from you. You may not attack the unready, the just, or the innocent.<br>\
+	You earn favor by getting others to join the crusade, and you may spend favor to announce a battle, bypassing some conditions to attack."
+	var/list/guilty = list() //list of guilty people
+
+/datum/religion_sect/honorbound/on_conversion(mob/living/new_convert)
+	RegisterSignal(new_convert, COMSIG_HUMAN_EARLY_UNARMED_ATTACK, .proc/unarmed_honor_check)
+	RegisterSignal(new_convert, COMSIG_ATOM_HULK_ATTACK, .proc/hulk_honor_check)
+	RegisterSignal(new_convert, COMSIG_MOB_ITEM_ATTACK, .proc/itemattack_honor_check)
+
+/datum/religion_sect/honorbound/proc/unarmed_honor_check(mob/living/carbon/human/honorbound_human, atom/target, proximity)
+	SIGNAL_HANDLER
+	if(!isliving(target))
+		return
+	var/mob/living/target_creature = target
+	if(!is_honorable(honorbound_human, target_creature))
+		return COMPONENT_CANCEL_ATTACK_CHAIN
+
+/datum/religion_sect/honorbound/proc/hulk_honor_check(atom/target, mob/living/carbon/human/honorbound_human)
+	SIGNAL_HANDLER
+	if(!isliving(target))
+		return
+	var/mob/living/target_creature = target
+	if(!is_honorable(honorbound_human, target_creature))
+		return COMPONENT_CANCEL_ATTACK_CHAIN
+
+/datum/religion_sect/honorbound/proc/itemattack_honor_check(atom/item, atom/target, mob/living/carbon/human/honorbound_human)
+	SIGNAL_HANDLER
+	if(!isliving(target))
+		return
+	var/mob/living/target_creature = target
+	if(!is_honorable(honorbound_human, target_creature))
+		return COMPONENT_CANCEL_ATTACK_CHAIN
+
+/datum/religion_sect/honorbound/proc/is_honorable(mob/living/carbon/human/honorbound_human, mob/living/target_creature)
+	var/is_guilty = (target_creature in guilty)
+	//THE UNREADY (Applies over ANYTHING else!)
+	if(target_creature.IsSleeping() || HAS_TRAIT(target_creature, TRAIT_RESTRAINED))
+		to_chat(honorbound_human, "<span class='warning'>There is no honor in attacking the <b>unready</b>.</span>")
+		return FALSE
+	//THE JUST (Applies over guilt except for med, so you best be careful!)
+	if(ishuman(target_creature))
+		var/mob/living/carbon/human/target_human
+		var/job = target_human.mind?.assigned_role
+		if(job in GLOB.security_positions || job == "Chaplain")
+			to_chat(honorbound_human, "<span class='warning'>There is nothing rightous in attacking the <b>just</b>.</span>")
+			return FALSE
+		if(job in GLOB.medical_positions)
+			to_chat(honorbound_human, "<span class='warning'>If you truly think this healer is not <b>innocent</b>, declare them guilty.</span>")
+			return FALSE
+	//THE INNOCENT
+	if(!is_guilty)
+		to_chat(honorbound_human, "<span class='warning'>There is nothing rightous in attacking the <b>innocent</b>.</span>")
+		return FALSE
+
+#define MINIMUM_YUCK_REQUIRED 5
+
+/datum/religion_sect/maintenance
```

Comment:
chap doesnt even get maint access

Reply:
I need to give a little bit of a quest to the chap, also new followers may

Reply:
The chap can just tunnel in like the maint rat he will become.